apps/web-mzima-client/src/app/app.component.spec.ts

```diff
@@ -17,13 +17,6 @@ describe('AppComponent', () => {
     expect(app).toBeTruthy();
   });
 
-  it(`should have as title 'platform-client'`, () => {
-    const fixture = TestBed.createComponent(AppComponent);
-    const app = fixture.componentInstance;
-
-    expect(app.title).toEqual('platform-client');
-  });
-
```

Comment:
@MugoBrian Please leave the tests for now since that is a different ball game. We will address this, but not now.
So just add the test code back for now.

Reply:
Ok I will. Actually @kvengerov instructed me to do so.

Reply:
yeah, @Ifycode  tests will fail if you make those changes only in app component, so probably we should adjust test files too.